src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -240,7 +239,7 @@ npm install aws-amplify
 ```
 
 In your app's entry point (i.e. **App.js**, **index.js**, **_app.js**, or **main.js**), import and load the configuration file:
-
+ 
```

Comment:
We may want to move this callout up since this was added to the Existing Resources tab. If that was an error and we remove that then no change in place here.

Reply:
commented the same - agreed.